includes/class-wordpress-template-integration.php

```diff
@@ -160,15 +160,20 @@ private function in_the_loop() {
 	}
 
 	public function display_view_in_content( $content = '' ) {
```

Comment:
@stephywells, this fix is a bit tricky, and I need to explain it. The `display_view_in_content` method is called multiple times in the Astra theme, and since we have the following condition, the term page is not displayed correctly:

```php
if ( $this->displayed ) {
	return '';
}
```

My solution:
1. I prevented the execution of the following code on the tax page by adding the following condition to remove the filter:

```php
if ( ! $is_tax ) {
	remove_filter( 'the_content', array( $this, 'display_view_in_content' ), 999 );
}
```

2. If we are on a tax page, the `$this->end_query();` code should prevent the `display_view_in_content` method from being executed again, which it does in all themes except Astra. I am sure that the Astra theme is incorrectly working with `$wp_query`, which causes this problem. I updated our condition as follows:

```php
if ( $this->displayed && ! $is_tax ) {
	return '';
}
```

In fact, when we are not on a tax page and this code is executed again, we should return an empty string, but on a tax page, we should prevent it so that `$this->end_query();` can work correctly.

Reply:
@stephywells, here are screenshots of the before and after of this change on the Astra theme:

### Before:
![bd local_business-directory_wpbdm-category_hardware_ (1)](https://user-images.githubusercontent.com/69119241/229653107-c6daecf9-0c0e-47e1-86b6-24efadafa40f.png)


### After:
![bd local_business-directory_wpbdm-category_hardware_](https://user-images.githubusercontent.com/69119241/229652904-be29543b-d192-4730-adc2-e3f07ad04ea5.png)